msteams-platform/bots/how-to/conversations/send-proactive-messages.md

```diff
@@ -54,11 +54,28 @@ Create the conversation, after you have the user or channel information.
 
 ## Create the conversation
 
-Create the conversation if it doesn't exist or you don't know the `conversationId`. Create the conversation only once and store the `conversationId` value or `conversationReference` object.
+You can create the conversation if it doesn't exist or you don't know the `conversationId`. Create the conversation only once and store the `conversationId` value or `conversationReference` object.
+
+To create a conversation, you need `userId`, `tenantId`, and `serviceUrl`.
+
+For `serviceUrl`, use the value from an incoming activity triggering the flow or one of the global service URLs. The following is an example for the `serviceUrl`:
+
+```csharp
+const serviceUrl = 'https://smba.trafficmanager.net/uk/';
+        `MicrosoftAppCredentials.trustServiceUrl(serviceUrl);
```

Comment:
this doesn't look like valid C# code

Reply:
The valid code should have " instead of ' and upper case T. Also using "var" or "string" instead of "const" might be better. 

var serviceUrl = "https://smba.trafficmanager.net/uk/";
MicrosoftAppCredentials.TrustServiceUrl(serviceUrl);

I also noticed that the .TrustServiceUrl() method is obsolete. Do we still want to include it? @aosolis  @tacri